build_stabilize.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+stabilize () {
+  file=$1
+  go install github.com/google/oss-rebuild/cmd/stabilize@dbdf78a8d293311c42192d40f72db0e54b63334b
+  ~/go/bin/stabilize -infile $file -outfile $file.stabilized
+  if [ $? -ne 0 ]; then
+    echo "Stabilize failed for $file"
+    exit 1
+  fi
+  echo "Result: $file.stabilized"
+}
+
+# path to .buildcompare file
+compare=$1
+# relative path to source code, usually buildcache/${artifactId}
+builddir=$2
+
+diffoscope_file=$(dirname ${compare})/$(basename ${compare} .buildcompare).oss-rebuild.diffoscope
+count="$(grep "^# diffoscope" ${compare} | wc -l)"
+
+[ $count -eq 0 ] && echo "No diffoscope command listed in $compare" && echo "oss_rebuild_ok=0" >> $compare && exit
+
+echo -e "saving build diffoscope file to \033[1m${diffoscope_file}\033[0m for $count issues"
```

Comment:
I'm not convinced about doing diffoscope in this script: this script is about creating stabilized content when necessary and seeing if it is sufficient